build.go

```diff
@@ -37,6 +37,12 @@ type Executable interface {
 func Build(entries EntryResolver, dependencies DependencyManager, logger scribe.Emitter, clock chronos.Clock, gem Executable) packit.BuildFunc {
 	return func(context packit.BuildContext) (packit.BuildResult, error) {
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
+		if s, ok := os.LookupEnv("BP_LOG_LEVEL"); ok {
+			logger.Logger = logger.Logger.WithLevel(s)
```

Comment:
I would expect this to be changed in `run` at the top level instantiation.

Reply:
So like `scribe.NewEmitter(os.Stdout).WithLevel(os.Getenv("BP_LOG_LEVEL"))`

Reply:
I have a few questions about this.
First, I can't directly call `WithLevel()` on the `scribe.Emitter` type unfortunately. I can only call it on the `Emitter`'s `Logger` type. Is this expected or is there a need to add a patch change to packit?

Second, I think it would be helpful to denote the acknowledge the environment variable has set, like I have in the code now. If I move the logger set up to `run/main.go`, I'm not sure of a straightforward way of denoting this. Do you know of a good way to accomplish this or is a change also needed to packit?

Reply:
> First, I can't directly call `WithLevel()` on the `scribe.Emitter` type unfortunately. I can only call it on the `Emitter`'s `Logger` type. Is this expected or is there a need to add a patch change to packit?

Ahhhhh now that I am looking at the function calls that does make sense. It may be worth adding a function to the `Emitter` but I think that can be decided later.

 > Second, I think it would be helpful to denote the acknowledge the environment variable has set, like I have in the code now. If I move the logger set up to run/main.go, I'm not sure of a straightforward way of denoting this. Do you know of a good way to accomplish this or is a change also needed to packit?

I am of two minds about this. On the one hand the user has to set the log level very intentionally through the environment variable so I am not sure how much feedback they need from that as well. On the other hand the point of the setting to to give a much information as possible. I think that at this point if another Ruby maintainer gives you the green light I wouldn't be upset but I think I would keep thinking about this interface for a little while.

Reply:
@ForestEckhardt to point number 2, I had the same thought. Might not be a huge deal, since they should be able to tell that the logs are more verbose.